src/pages/phoneBook/phoneBook.js

```diff
@@ -0,0 +1,17 @@
+import Container from "../../Container/container";
+import Navigation from "../../Navigation/navigation";
```

Comment:
Do you like these imports?
"Container/container"

You can rename navigation.js => index.js and remove the extra name.
by the way, all components must be name from the Upper letter => Container.jsx

And also all your components has type ".js" not ".jsx", please fix it